source/client-side-encryption/client-side-encryption.rst

```diff
@@ -2,16 +2,8 @@
 Client Side Encryption
 ======================
 
-:Title: Client Side Encryption
-:Author: Kevin Albertson
-:Spec Lead: Jeff Yemin
-:Approvers: Bernie Hackett, David Golden, David Storch
-:Advisory Group: A\. Jesse Jiryu Davis, Kenn White, Scott L'Hommedieu, Mark Benvenuto, Bernie Hackett, Samantha Ritter, Matt Broadstone
 :Status: Accepted
-:Type: Standards
 :Minimum Server Version: 4.2 (CSFLE), 6.0 (Queryable Encryption)
-:Last Modified: 2022-09-26
-:Version: 1.12.0
 
 .. _lmc-c-api: https://github.com/mongodb/libmongocrypt/blob/master/src/mongocrypt.h.in
```

Comment:
Suggest removing the version from the `versionadded` directives in this file, since the version is no longer defined.

Reply:
Ah, I had no idea that was used (GitHub doesn't apply any special rendering, but I imagine @vector-of-bool had some nice formatting when building with Sphinx in https://github.com/mongodb/specifications/commit/52759c75816a89f8ae6430b22cec9571451e6c7d).

Would have been nice if we started using that in the unified test format spec from the beginning, but it's probably a bit late for that now. And we do have schema changes highlighted in the changelog there, so probably not really necessary.

Reply:
Removed those macros in https://github.com/mongodb/specifications/pull/1311/commits/cc51ecc2590755dfb0ad6c5e08e330cbf2e083e4